### DISCUSSION TITLE
v3 feature completion

### DISCUSSION THREAD

**Discussion by kasperk81** [xunit/xunit#2940] [General]
v3 is receiving dozens of new features and enhancements https://github.com/xunit/xunit/issues/2133. can some of the features be left out for v3.1 or v4 to expedite its release, sometime this year?

**Comment by bradwilson** [comment]
Two thoughts.

First, the list is aspirational. Not everything on the list will be done by 3.0.

Second, the major issue that's holding back a first wide pre-release is https://github.com/xunit/xunit/issues/2337 and it's what I'm working on right now.
